Index: src/frontends/qt4/FindAndReplace.h
===================================================================
--- src/frontends/qt4/FindAndReplace.h	(revision 37858)
+++ src/frontends/qt4/FindAndReplace.h	(revision 37859)
@@ -53,11 +53,15 @@
 	);
 
 	/// Perform the scope-related buffer switch while searching
-	void findAndReplaceScope(FindAndReplaceOptions & opt);
+	bool findAndReplaceScope(FindAndReplaceOptions & opt, bool replace_all);
 
 	/// Collect options from the GUI elements, then perform the search
-	void findAndReplace(bool backwards, bool replace);
+	bool findAndReplace(bool backwards, bool replace, bool replace_all = false);
 
+	bool findAndReplace(bool casesensitive, bool matchword, bool backwards,
+		bool expandmacros, bool ignoreformat, bool replace,
+		bool keep_case, bool replace_all);
+
 	bool eventFilter(QObject *obj, QEvent *event);
 
 	void virtual showEvent(QShowEvent *ev);
